plugins/lookup/tss.py

```diff
@@ -196,13 +207,13 @@ def from_params(**server_parameters):
         else:
             return TSSClientV0(**server_parameters)
 
-    def get_secret(self, term):
+    def get_secret(self, term, query_params):
         display.debug("tss_lookup term: %s" % term)
 
         secret_id = self._term_to_secret_id(term)
         display.vvv(u"Secret Server lookup of Secret with ID %d" % secret_id)
 
-        return self._client.get_secret_json(secret_id)
+        return self._client.get_secret_json(secret_id, query_params)
```

Comment:
This should make it compatible with older versions (assuming the usre does not specify `comment`):
```suggestion
        if query_params:
            return self._client.get_secret_json(secret_id, query_params)
        return self._client.get_secret_json(secret_id)
```